src/rt/cover.c

```diff
@@ -1304,37 +1318,62 @@ static void cover_print_hierarchy_summary(FILE *f, cover_stats_t *stats, ident_t
 
    fprintf(f, "   </tr>\n");
 
+   float perc_stmt = 0;
```

Comment:
```suggestion
   float perc_stmt = 0.0f;
```

(Or 0.0 if you're going to use `double`.)